modules/nodes-vsphere-machine-set-scaling-static-ip.adoc

```diff
@@ -0,0 +1,193 @@
++
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  annotations:
+    machine.openshift.io/memoryMb: "8192"
+    machine.openshift.io/vCPU: "4"
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+  name: <infrastructure_id>-<role>
+  namespace: openshift-machine-api
+spec:
+  replicas: 0
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
+  template:
+    metadata:
+      labels:
+        ipam: "true"
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
+    spec:
+      lifecycleHooks: {}
+      metadata: {}
+      providerSpec:
+        value:
+          apiVersion: machine.openshift.io/v1beta1
+          credentialsSecret:
+            name: vsphere-cloud-credentials
+          diskGiB: 120
+          kind: VSphereMachineProviderSpec
+          memoryMiB: 8192
+          metadata: {}
+          network:
+            devices:
+            - addressesFromPools: <1>
+              - group: ipamcontroller.openshift.io
+                name: static-ci-pool
+                resource: IPPool
+              nameservers:
+              - "192.168.204.1" <2>
+              networkName: qe-segment-204
+          numCPUs: 4
+          numCoresPerSocket: 2
+          snapshot: ""
+          template: rvanderp4-dev-9n5wg-rhcos-generated-region-generated-zone
+          userDataSecret:
+            name: worker-user-data
+          workspace:
+            datacenter: IBMCdatacenter
+            datastore: /IBMCdatacenter/datastore/vsanDatastore
+            folder: /IBMCdatacenter/vm/rvanderp4-dev-9n5wg
+            resourcePool: /IBMCdatacenter/host/IBMCcluster//Resources
+            server: vcenter.ibmc.devcluster.openshift.com
+----
+<1> Specifies an IP pool, which lists a static IP address or a range of static IP addresses. The IP Pool can either be a reference to a custom resource definition (CRD) or a resource supported by the `IPAddressClaims` resource handler. The machine controller accesses static IP addresses listed in the machine set's configuration and then allocates each address to each machine. 
+<2> Lists a nameserver. You must specify a nameserver for nodes that receive static IP address, because the Dynamic Host Configuration Protocol (DHCP) network configuration does not support static IP addresses. 
+
+. Scale the machine set by entering the following commands in your `oc` CLI:
++
+[source, terminal]
+----
+$ oc scale --replicas=2 machineset <machineset> -n openshift-machine-api
+----
++
+Or:
```

Comment:
SNE query: I'm a bit confused with the steps for this procedure. I might have stumbled into using a mixture of manual steps and machine controller automated steps? WDYT? 

Maybe I should state both routes? Or does a blend exist?

I mentioned the external controller as I seen it referenced in [this GH page](https://github.com/openshift/enhancements/blob/5d8751c23dc39a878b3b73d64fe44745aceb25d9/enhancements/network/static-ip-addresses-vsphere.md#scaling-new-nodes-with-machinesets).

Reply:
It is very much a blend.  The machine API controller creates a resource that either some external controller(i.e. the GH page you linked) or an admin handles via `oc` commands.  My intention was to demonstrate how this can be done with `oc` so that an administrator can create the necessary automation based on what they want or need to use.

Reply:
I would keep it around as a note.

Reply:
Thanks, @rvanderp3 . I converted the paragraph to a note and prepended it with "By default". Having a string of notes can be distracting from a docs perspective, so I converted the other note to an optional step. WDYT?

![Screenshot from 2023-08-15 12-09-02](https://github.com/openshift/openshift-docs/assets/57954076/b581afb5-237f-4832-9ad1-257c62c6d73e)